### ISSUE TITLE
Keyboard-interactable element definition should not define target of interaction

### ISSUE THREAD

**Issue by andreastt** [w3c/webdriver#886]
In chapter [11. Elements](https://w3c.github.io/webdriver/webdriver-spec.html#dfn-keyboard-interactable-element#h-elements), section [11.1 Element Interactability](https://w3c.github.io/webdriver/webdriver-spec.html#dfn-keyboard-interactable-element#h-element-interactability):
> A keyboard-interactable element is defined to be any element that has a focusable area, or is the body element. In the case of the body element all keyboard interactions must be targeted to its ownerDocument. 

It does not make sense for the _keyboard-interactable element_ definition to contain normative language on which elements should be the target of the interaction.

**Comment by shs96c** [comment]
How would you suggest this be phrased? I'm guessing you're suggesting that we omit the text entirely?

**Comment by shs96c** [comment]
Assigning to you for now. Please unassign yourself if you'd like to.

**Comment by andreastt** [comment]
I tried reading up on the definition of [focusable area](https://html.spec.whatwg.org/#focusable-area) in the HTML spec, but I’m not sure I fully understand it yet.

When I filed this issue, I was mostly thinking about the fact that we can’t talk about what the target is for keyboard interaction in the context of defining which kind of elements are interactable by keyboard. Which element will receive the DOM event is inferred by [‘dispatch a keyDown action’](https://w3c.github.io/webdriver/webdriver-spec.html#dfn-dispatch-a-keydown-action), which is responsible for the interaction.
